core/src/main/kotlin/org/evomaster/core/problem/rest/service/fitness/AbstractRestFitness.kt

```diff
@@ -1316,6 +1325,37 @@ abstract class AbstractRestFitness : HttpWsFitness<RestIndividual>() {
         }
     }
 
+    private fun handleSQLiCheck(
+        individual: RestIndividual,
+        actionResults: List<ActionResult>,
+        fv: FitnessValue
+    ) {
+        if (!config.isEnabledFaultCategory(DefinedFaultCategory.XSS)) {
```

Comment:
xss???  also need a check on SQLi, which would be off by default, as still experimental. 
however, maybe we should avoid long checks like:

`!config.xss || !config.isEnabledFaultCategory(DefinedFaultCategory.XSS)`

where the check for `config.xss` (and new `!config.sqli`) should be done inside `isEnabledFaultCategory`. eg something like:

```
    fun isEnabledFaultCategory(category: FaultCategory) : Boolean{
       if(category==DefinedFaultCategory.XSS && !xss){
          return false;
       }  
       return category !in getDisabledOracleCodesList()
    }
```